docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4544,7 +4562,23 @@ four replicas of an ARP packet. These replicas will appear in the egress
 pipeline as independent packets with egress port set to PSA device port numbers
 corresponding to SDN port numbers 5, 12, 18 and 24. For more discussion on the
 translation between SDN ports and PSA device ports, refer to the
-[PSA Metadata Translation](#sec-translation-of-port-numbers) section.
+[PSA Metadata Translation](#sec-translation-of-port-numbers) section. Each
+replica can optionally have a list of backup replicas used as fallback ports
+for multicast. When the primary port (replica) goes down, the system uses the
+first backup replica whose port is up. Whenever the primary port goes back up,
```

Comment:
"Whenever the primary port goes back up, the system will use the primary port again."

A similar thing happens when we're currently using back up 2, but then back up 1's port comes back up, right? Can we generalize this a bit to capture that as well?